nextflow_schema.json

```diff
@@ -597,54 +642,78 @@
                     "description": "Path to ASCAT GC content correction file.",
                     "help_text": "If you use AWS iGenomes, this has already been set for you appropriately."
                 },
                 "ascat_loci_rt": {
                     "type": "string",
                     "fa_icon": "fas fa-file",
+                    "format": "file-path",
+                    "pattern": "^\\S+\\.zip$",
+                    "exists": true,
                     "description": "Path to ASCAT RT (replictiming) correction file.",
                     "help_text": "If you use AWS iGenomes, this has already been set for you appropriately."
                 },
                 "bwa": {
                     "type": "string",
+                    "format": "directory-path",
+                    "exists": true,
                     "fa_icon": "fas fa-copy",
                     "description": "Path to BWA mem indices.",
                     "help_text": "If you wish to recompute indices available on igenomes, set `--bwa false`.\n\n> **NB** If none provided, will be generated automatically from the FASTA reference. Combine with `--save_reference` to save for future runs.\n\nIf you use AWS iGenomes, this has already been set for you appropriately."
                 },
                 "bwamem2": {
                     "type": "string",
+                    "format": "directory-path",
+                    "exists": true,
                     "fa_icon": "fas fa-copy",
                     "description": "Path to bwa-mem2 mem indices.",
                     "help_text": "If you use AWS iGenomes, this has already been set for you appropriately.\n\nIf you wish to recompute indices available on igenomes, set `--bwamem2 false`.\n\n> **NB** If none provided, will be generated automatically from the FASTA reference, if `--aligner bwa-mem2` is specified. Combine with `--save_reference` to save for future runs."
                 },
                 "chr_dir": {
                     "type": "string",
+                    "format": "path",
+                    "exists": true,
                     "fa_icon": "fas fa-folder-open",
                     "description": "Path to chromosomes folder used with ControLFREEC.",
                     "help_text": "If you use AWS iGenomes, this has already been set for you appropriately."
                 },
                 "dbsnp": {
                     "type": "string",
+                    "format": "file-path",
+                    "pattern": "^\\S+\\.vcf\\.gz$",
+                    "exists": true,
                     "fa_icon": "fas fa-file",
                     "description": "Path to dbsnp file.",
                     "help_text": "If you use AWS iGenomes, this has already been set for you appropriately."
                 },
                 "dbsnp_tbi": {
                     "type": "string",
+                    "format": "file-path",
+                    "pattern": "^\\S+\\.vcf\\.gz\\.tbi$",
```

Comment:
why are you using here the full vcf.gz.tbi? Above it is just .tbi

Reply:
Ah that's probably a small inconsistency on my part :) I'll fix it